README.md

```diff
@@ -1,2 +1,41 @@
 # RunAway
-An infinity runner type Android game built on Unity.
+
+## Team members
+* Amartya Jayawant Patil
+* Devayani Shivankar
+* Vaishnavi Jarwal
+* Navya Sharma
+* Reshmika Sreenath Nambiar
+
+## Mentor
+* Shubhankar Gupta
+
+## Description
+RunAway is an endless runner Android game built on Unity. The player controls a mermaid who turns into a human on land. They have to dodge obstacles and collect coins to
+increase their score. The game ends when the player crashes into an obstacle. It is an infinity runner-type game with the camera following the character. The character can move left or right to dodge obstacles. They can also jump. The farther the player runs, the more the score. There is also a login feature through which the player's score gets stored. Players may also choose to view their Stats such as high scores,
+the longest distance run, or the total coins collected.
+
+* GitHub repo link: [Link to repository](https://github.com/DevayaniShivankar/RunAway)
+* Drive link: [Drive link here](https://drive.google.com/)
+
+## Technology stack
+
+1. C#
+2. Unity
+3. Firebase
```

Comment:
Mention these points as:
1. Scripting Language: C#
2. Platform: Unity (2020.3.3f1)
3. Database: Firebase #

Reply:
I've added this now.